modynclient/client/client.py

```diff
@@ -169,3 +170,24 @@ def poll_pipeline_status(self) -> None:
             logger.info(f"Pipeline <{self.pipeline_id}> not found.")
         else:
             logger.error(f"unknown pipeline status {json.dumps(res, sort_keys=True, indent=2)}")
+
+
+def run_multiple_pipelines(
+    client_config: ModynClientConfig,
+    pipeline_configs: list[ModynPipelineConfig],
+    start_replay_at: int = 0,
+    stop_replay_at: int | None = None,
+    maximum_triggers: int | None = None,
+) -> None:
+    logger.info("Start running multiple experiments!")
+    logger.info(f"{client_config.model_dump()}")
```

Comment:
Debug dump?

Reply:
It's also done in the entrypoint script, I'll remove it.

Reply:
thx!